From 116de6895c307ceee798a92d801cc14f41dd1dd1 Mon Sep 17 00:00:00 2001
From: Patrice Chotard <patrice.chotard@foss.st.com>
Date: Tue, 6 Jul 2021 16:49:42 +0200
Subject: [PATCH] arm64: defconfig: Add Octo Memory Manager support

Enable Octo Memory Manager relative flag.

Signed-off-by: Patrice Chotard <patrice.chotard@foss.st.com>
Change-Id: Ibe9e9fb32efb0649f7796d36cb2f1778dc3fc3f7
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/307844
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -40,6 +40,7 @@ CONFIG_TEE_REMOTEPROC=y
 CONFIG_UBIFS_FS=y
 CONFIG_STM32_HYPERBUS=y
 CONFIG_STM32_OMI=y
+CONFIG_STM32_OMM=y
 CONFIG_USB_HIDDEV=y
 CONFIG_USB_DWC3_STM32=y
 CONFIG_USB_STM32_USBH=y
